docs/source/about/team.rst

```diff
@@ -50,34 +52,75 @@ Core Developers
 
    * - Name
      - GitHub ID
+     - Current affiliation of contribution
+     - Affiliation of primary role
```

Comment:
What does affiliation of primary role mean? Is 'role' referring to 'sktime governance roles' or to roles outside of sktime?

Reply:
Never mind, just saw the explanation at the bottom. Maybe move the explations of the columns above the table?

Reply:
I've added a better explanation, and the header is also clearer now, I hope?

Reply:
I don't think you have pushed the commits!

(Also, github has deleted a comment I made. In it, I suggested moving the explanation of the columns above the table rather than after the table.)

Reply:
??? I thought I did. You might be looking at an outdated version? Or I'm confused.

Reply:
>(Also, github has deleted a comment I made. In it, I suggested moving the explanation of the columns above the table rather than after the table.)

I don't think GitHub deletes comments. There might be something wrong with your browser cache or internet connectio?.